### PR TITLE
Simplify how lab page is rendered

### PR DESCRIPTION
**Pull request by k-nut**
- Show all content without accordion
- Remove al to of bootstrap classes for a more
  simple layout

### CODE REVIEW

File: themes/codefor-theme/layouts/labs/single.html [hunk 1]
```diff
@@ -1,85 +1,44 @@
...
-                    </div>
-                </div>
-            </div>
-
-		</div>
-	</div>
+<div class="lab-content">
+    <h1 class="d-flex align-items-center flex-wrap justify-content-center">
+        <img src="/labs/hexagon/CodeFor-{{ .File.BaseFileName }}.svg" alt="Logo {{ .Title }}">
+        <div class="flex-grow-1"> {{.Title}} </div>
```
Comment (ubergesundheit) [inline themes/codefor-theme/layouts/labs/single.html:7]:
```suggestion
        <div class="flex-grow-1 text-center"> {{.Title}} </div>
```

**Comment by ubergesundheit** [comment]
Really nice!

I would do some addtional changes:

- Add the fancy brackets around the logo and lab name
- Add the [`fileExists` check for the lab logo](https://github.com/okfde/codefor.de/pull/216/files#diff-e34662544e000d6f84946a49e1fe48200945bf6f586f8ffa42417a1b5b751050R3)

**Comment by k-nut** [comment]
@ubergesundheit done ✅ 
Please take another look.

**Review by ubergesundheit — APPROVED** [review]
Lookin good 👌

**Pull request merged**